tests/test_org_sessions_api.py

```diff
@@ -0,0 +1,120 @@
+import itertools
+from datetime import datetime, timedelta
+from typing import Any, Callable, Tuple, Union
+
+import pytest
+import pytz
+import simplejson as json
+
+from snuba import settings
+from snuba.consumers.types import KafkaMessageMetadata
+from snuba.datasets.storages import StorageKey
+from snuba.datasets.storages.factory import get_writable_storage
+from tests.base import BaseApiTest
+from tests.helpers import write_processed_messages
+
+
+class TestOrgSessionsApi(BaseApiTest):
+    @pytest.fixture
+    def test_entity(self) -> Union[str, Tuple[str, str]]:
+        return "org_sessions"
+
+    @pytest.fixture
+    def test_app(self) -> Any:
+        return self.app
+
+    @pytest.fixture(autouse=True)
+    def setup_post(self, _build_snql_post_methods: Callable[[str], Any]) -> None:
+        self.post = _build_snql_post_methods
+
+    @pytest.fixture(scope="class")
+    def get_project_id(self, request: object) -> Callable[[], int]:
+        id_iter = itertools.count()
+        next(id_iter)  # skip 0
+        return lambda: next(id_iter)
+
+    def setup_method(self, test_method: Any) -> None:
+        super().setup_method(test_method)
+
+        # values for test data
+        self.minutes = 180
+        self.skew = timedelta(minutes=self.minutes)
+        self.started = datetime.utcnow().replace(
+            minute=0, second=0, microsecond=0, tzinfo=pytz.utc
+        )
+
+        self.storage = get_writable_storage(StorageKey.SESSIONS_RAW)
+
+    def generate_session_events(self, project_id: int) -> None:
+        processor = self.storage.get_table_writer().get_stream_loader().get_processor()
+        meta = KafkaMessageMetadata(
+            offset=1, partition=2, timestamp=datetime(1970, 1, 1)
+        )
+        template = {
+            "session_id": "00000000-0000-0000-0000-000000000000",
+            "distinct_id": "b3ef3211-58a4-4b36-a9a1-5a55df0d9aaf",
```

Comment:
I would maybe change these so they're not the same as the other session test values. Just in case.